sources/web/datalab/static/settings.html

```diff
@@ -4,15 +4,23 @@
   <meta charset="utf-8">
 </head>
 <body>
-  Theme:
   <div>
+    Theme:
     <div class="radio">
       <label><input type="radio" id="lightThemeRadioOption" checked="">Light Theme</label>
     </div>
     <div class="radio">
       <label><input type="radio" id="darkThemeRadioOption" checked="">Dark Theme</label>
     </div>
   </div>
+  <div>
+    Auto-shutdown idle timeout interval (0 to disable):
```

Comment:
That works too I think. We have a lot of room here though, this is why I proposed this design. Up to you. :)